docs/dcp2_system_design.rst

```diff
@@ -989,6 +989,20 @@ Broad Institute).
 Project-level matrices
 ======================
 
+In the context of the HCA DCP, a matrix is a two-dimensional structure relating
+individual cell identities (or an approximation thereof) to properties of the
+the transcriptome of those cells e.g., genes expressed in the transcriptome,
+number of those cells.
+
+A *project-level* matrix is a matrix that covers all or a large subset of the
+cells involved in the experiments for a particular project. A project-level
+matrix is *stratified* if it is partitioned over separate files, using certain
+criteria on the properties of the experiments in a project or the cells studied
+in those experiments.
+
+A matrix from a contributor or any other source external to the DCP is refered
+to as a *contributor-generated* matrix.
+
 
 Contributor-generated matrices (CGMs)
```

Comment:
Agreed but that would be fairly disruptive because we the term "CGM" has been pretty much established. We can change the definition though to include these supporting files.

Reply:
Addressed.